rescript

```diff
@@ -2,71 +2,27 @@
 //@ts-check
 "use strict";
 
-/* This script is supposed to be running in project root directory
+/**
+ * This script is supposed to be running in project root directory
  * It matters since we need read .sourcedirs(location)
  * and its content are file/directories with regard to project root
  */
 
-var child_process = require("child_process");
-var os = require("os");
-var path = require("path");
-var fs = require("fs");
 var bsc_exe = require("./scripts/bin_path").bsc_exe;
 var rescript_exe = require("./scripts/bin_path").rescript_exe;
+var bsb = require("./scripts/rescript_bsb");
 
-var LAST_SUCCESS_BUILD_STAMP = 0;
 var cwd = process.cwd();
-var lockFileName = path.join(cwd, ".bsb.lock");
 process.env.BSB_PROJECT_ROOT = cwd;
 
-const isTtyError = process.stderr.isTTY;
-const isTtyStd = process.stdout.isTTY;
-
-let verbose = false;
-
-/**
- * @time{[number,number]}
- */
-let startTime;
-function updateStartTime() {
-  startTime = process.hrtime();
-}
-function updateFinishTime() {
-  const diff = process.hrtime(startTime);
-  return diff[0] * 1e9 + diff[1];
-}
-
-/**
- *
- * @param {string} file
- * @returns
- */
-function getWatchFiles(file) {
-  if (fs.existsSync(file)) {
-    return JSON.parse(fs.readFileSync(file, "utf8"));
-  } else {
-    return { dirs: [], generated: [] };
-  }
-}
-
-/**
- *
- * @param {*} str
- */
-function dlog(str) {
-  if (verbose) {
-    console.log(str);
-  }
-}
-
-var process_argv = process.argv;
-
 if (process.env.NINJA_ANSI_FORCED === undefined) {
   if (require("tty").isatty(1)) {
     process.env.NINJA_ANSI_FORCED = "1";
   }
 } else {
-  dlog(`NINJA_ANSI_FORCED: "${process.env.NINJA_ANSI_FORCED}"`);
+  if (process.argv.includes("-verbose")) {
```

Comment:
It used to be broken before. Should we keep the fix or remove the log?

Reply:
I would say keep the fix